resolver/Cargo.toml

```diff
@@ -29,6 +29,12 @@ thiserror = { workspace = true }
 
 [dev-dependencies]
 tracing-subscriber = { workspace = true }
+commonware-resolver = { path = ".", features = ["mocks"] }
```

Comment:
~Alternative proposal, I suppose, is just to remove this `mocks` feature? I think it is pretty small in practice (and I'm not sure we take this sort of granular approach elsewhere yet).~

Let's keep it and just see if we can avoid the test changes (I suspect this won't be the first time we need to do something like this).